api/everest.go

```diff
@@ -44,6 +44,10 @@ import (
 	"github.com/percona/everest/public"
 )
 
+const (
+	propertySkipEverestAuthnSwagger = "x-everest-skip-authn"
+)
+
```

Comment:
I was wondering if there is a native openapi way to specify the security schema without a need to define the user-defined constant. And there is https://swagger.io/docs/specification/authentication/ 
So theoretically we could define the bearer auth as a default one for the API and then override the security settings to empty for the endpoints which don't require authentication. And then retrieve this information in the code to figure out if the method needs to be skipped.   

Not sure if it's worth to put more effort on the topic, so leaving it up to you, just wanted to bring it to consideration